### PR TITLE
e2e: first group

### LINKED ISSUE #12895: Add to upgrade e2e job options to run separate test cases
# Problem

As uppgrade e2e are fragile and flaky, option to rerun end-to-end-tests/job/status-app-upgrade/ with TCs will be useful

## Implementation

add param to job

### PR DESCRIPTION
**Pull request by churik**
1) Added 1t bunch of e2e for public chat.
The whole test group passes in ~11 min.
Based on https://ethstatus.testrail.net/index.php?/suites/view/48&group_by=cases:section_id&group_order=asc&display_deleted_cases=0&group_id=50654 (Public chat group)

2) Fix https://github.com/status-im/status-react/issues/12895

3) Updated jobs to make possible run based on name or class name (keyword expression)

### CODE REVIEW

**Review by yevh-berdnyk — APPROVED** [review]

**Comment by churik** [comment]
Launching failed e2e against release 1.17

File: ci/tests/Jenkinsfile.e2e-nightly [hunk 1]
```diff
@@ -56,8 +56,8 @@ pipeline {
                 --rerun_count=2 \
                 --testrail_report=True \
                 -m testrail_id \
-                -m "not upgrade" \
-                --network=${params.NETWORK} \
+                -m \"not upgrade\" \
+                -k \"${params.KEYWORD_EXPRESSION}\" \
```
Comment (jakubgs) [inline ci/tests/Jenkinsfile.e2e-nightly:60]: I'm not sure if this escaping of double quotes is necessary here.

**Review by jakubgs — COMMENTED** [review]
Lgtm, aside from maybe the quote escaping.

File: ci/tests/Jenkinsfile.e2e-nightly (hunk 1 above)
Comment (churik) [reply ci/tests/Jenkinsfile.e2e-nightly:60]: in -m  it is not, in -k this is necessary, checked 
So I made it the same for consistency

Comment (jakubgs) [reply ci/tests/Jenkinsfile.e2e-nightly:60]: Cool.

**Pull request merged**